packages/components/src/components/primitives/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Toggle } from '.';
+import { boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Primitives/Toggle',
+  component: Toggle,
+};
+
+export const Playground = () => (
+  <Toggle
+    onStateChanged={action('on state changed')}
+    enabled={boolean('Enabled', false)}
+    disabled={boolean('Disabled', false)}
+  />
+);
```

Comment:
we should have both states in a component to allow chromatic to catch regressions

Reply:
Added default stories